src/util/config.rs

```diff
@@ -4,8 +4,10 @@
 /// Top-level config which holds ChannelHandshakeLimits and ChannelConfig.
 #[derive(Clone, Debug)]
 pub struct UserConfig {
+	/// Channel config that we propose on receiving a channel opening request
+	pub own_inbound_channel_config: ChannelHandshakeConfig,
 	/// Limits applied during channel creation.
-	pub channel_limits: ChannelHandshakeLimits,
+	pub peer_channel_config_limits: ChannelHandshakeLimits,
```

Comment:
Can you change the comment above to say something like "Limits applied to the peer's channel config parameters during channel creation".